### ISSUE TITLE
@ionic-super-tabs/core@6.0.3 question

### ISSUE THREAD

**Issue by EdwardEvad3rs** [zyra/ionic-super-tabs#398]
For this version, the active and inactive icon color can be changed by using .scss file but indicator color was not (only white color). (--st-indicator-color no effect at all)
In the previous version, indicator color can be changed but active and inactive icon color was not. (--st-base-color-active & --st-base-color-inactive no effect at all)

Is that any way to solve this?

**Comment by ihadeed** [comment]
Can you share how you're attempting to set those variables?

I noticed that variables must be set directly on the affected element for them to work. I think setting it on the `super-tab-button` should work, but you can assign it directly to `ion-label/icon`. This is definitely something that should be improved.

```scss
super-tab-button {
  // this should work
  --st-label-color-active: yellow;
 
  ion-label {
    // this should also work
    --st-label-color-inactive: green;
  }

  ion-icon {
    --st-icon-color-active: rgba(0,0,0,0.1);
  }
}
```

**Comment by EdwardEvad3rs** [comment]
this is how I set.

super-tab-button {
  --st-base-color-inactive: rgba(255,255,255,0.4);
  --st-base-color-active: var(--ion-color-secondary);
  --st-icon-color-active: var(--ion-color-secondary-tint);
  
}

super-tabs-toolbar {
  --st-indicator-color: var(--ion-color-secondary-tint);
}

my secondary color was red, and it still shows rgba(255,255,255,0.4) for both inactive and active.

**Comment by EdwardEvad3rs** [comment]
But I am trying your method, will update here after I try.

**Comment by EdwardEvad3rs** [comment]
Everything worked except the active indicator.

**Comment by malua** [comment]
I think the active indicator is overwritten by this in

> super-tab-indicator.component.scss:

 ```
 --st-indicator-height: 2px;
 --st-indicator-color: var(--ion-color-contrast, white);
  height: var(--st-indicator-height);
  background: var(--st-indicator-color);

```
and I think it should be like this.
first two lines removed and instead only these two lines:

```
height: var(--st-indicator-height, 2px);
background: var(--st-indicator-color, var(--ion-color-contrast, white));
```

I can provide a pull request if you'd like.

**Comment by ihadeed** [comment]
Thanks @malua!
That worked perfectly, I wasn't sure how to make CSS variables pass through elements.

Just updated all CSS variables to that format so they can be set in `variables.scss`/`:root`, a parent element, or the element itself. Fix is available in `^6.1.0`.
